daemon/oci_linux.go

```diff
@@ -809,7 +809,7 @@ func (daemon *Daemon) createSpec(c *container.Container) (retSpec *specs.Spec, e
 			s.Hooks = &specs.Hooks{
 				Prestart: []specs.Hook{{
 					Path: target,
-					Args: []string{"libnetwork-setkey", c.ID, daemon.netController.ID()},
+					Args: []string{"libnetwork-setkey", "-exec-root=" + daemon.configStore.GetExecRoot(), c.ID, daemon.netController.ID()},
```

Comment:
Ah, just realized this uses a single dash for the flag (`-exec-root`, not `--exec-root`) probably because it's using the standard `flag` library..

Should we only pass this option if it's not the default, or pass it in all situations? (not sure I have a real preference, just putting it here so that we can discuss)

Reply:
Alright; it was not a blocker, just a question, so let's get this merged 👍 thanks!